.github/workflows/test.yml

```diff
@@ -249,11 +274,21 @@ jobs:
           secret_token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Compile and run tests
-        uses: eProsima/eProsima-CI/multiplatform/colcon_build_test_flaky@v0
+        id: compile_and_test
+        uses: eProsima/eProsima-CI/multiplatform/colcon_build_test@feature/test-results
```

Comment:
Why not use `colcon_build_test_flaky`?

Reply:
While reviewing the eProsima-CI action I saw that `colcon_build_test_flaky` is exactly the same as `colcon_build_test` but whith come extra `ctest-args`. I'd prefer to use the same for both and add this extra configuration on the workflow

Reply:
Okay, we can remove it in the future.